src/ip.py

```diff
@@ -727,17 +727,24 @@ def do_neigh_flush(argv, af):
 def main(argv):
     # Detect Address family
     af = -1  # default / both
-    if argv and argv[0] == "-6":
+    if "-6" in argv:
         af = 6
-        argv.pop(0)
-    elif argv and argv[0] == "-4":
+        argv.remove("-6")
+    elif "-4" in argv:
         af = 4
-        argv.pop(0)
+        argv.remove("-4")
+    
+    # Ignore color option (-c --color)
+    for option in ["-c", "--c"]:
+        for arg in argv:
+            if arg.startswith(option):
+                perror("!Color option not available!")
```

Comment:
This message is somewhat tricky, but I agree printing to stderr is likely going to be okay.

I suggest: `perror("iproute2mac: Color option is not implemented")`